libdnf5-plugins/CMakeLists.txt

```diff
@@ -4,3 +4,7 @@ set(CMAKE_C_VISIBILITY_PRESET hidden)
 add_subdirectory("actions")
 add_subdirectory("python_plugins_loader")
 add_subdirectory("rhsm")
+
+if(WITH_APPSTREAM)
```

Comment:
Please add the subdirectory unconditionally and move the condition within the appstream/CMakeLists.txt like it is done e.g. for actions plugin.